src/twisted/newsfragments/10111.misc

```diff
@@ -0,0 +1 @@
+test on py3.10a5
```

Comment:
Can this be a feature news fragment like announcing that 3.10 is supported on Linux?